<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsFeedsMessages</class>
 <widget class="QWidget" name="SettingsFeedsMessages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>531</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="m_tabFeedsMessages">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_tabFeeds">
      <attribute name="title">
       <string>Feed fetching</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="m_checkUpdateAllFeedsOnStartup">
         <property name="text">
          <string>Fetch all articles on startup with initial delay of</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="TimeSpinBox" name="m_spinStartupUpdateDelay">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="m_checkAutoUpdate">
         <property name="text">
          <string>Auto-fetch articles for all feeds every</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="TimeSpinBox" name="m_spinAutoUpdateInterval">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="m_checkAutoUpdateOnlyUnfocused">
         <property name="text">
          <string>Only auto-fetch articles if application is unfocused</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cmbFastAutoUpdate">
         <property name="text">
          <string>Support very fast auto-fetching intervals (under 10 seconds)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Feed connection timeout</string>
         </property>
         <property name="buddy">
          <cstring>m_spinFeedUpdateTimeout</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="m_spinFeedUpdateTimeout">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Connection timeout is time interval which is reserved for downloading new messages for the feed. If this time interval elapses, then download process is aborted.</string>
         </property>
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>120000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="ArticleAmountControl" name="m_wdgArticleLimiting" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabFeedsList">
      <attribute name="title">
       <string>Feeds list</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Row height</string>
         </property>
         <property name="buddy">
          <cstring>m_spinHeightRowsFeeds</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="m_spinHeightRowsFeeds">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="suffix">
          <string notr="true"> px</string>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="m_gbFeedListFont">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Feed list font</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="m_lblFeedListFont">
              <property name="text">
               <string>Font preview</string>
              </property>
              <property name="buddy">
               <cstring>m_btnChangeFeedListFont</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="m_btnChangeFeedListFont">
              <property name="text">
               <string>&amp;Change font</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Article count format</string>
         </property>
         <property name="buddy">
          <cstring>m_cmbCountsFeedList</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="m_cmbCountsFeedList">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="HelpSpoiler" name="m_helpCountsFeedsFormat" native="true"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cbHideCountsIfNoUnread">
         <property name="text">
          <string>Hide article counts if there are no unread articles</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cbListsRestrictedShortcuts">
         <property name="text">
          <string>Allow only basic keyboard shortcuts for feed/article list</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="m_checkShowTooltips">
         <property name="text">
          <string>Display tooltips for feeds and articles</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cbStrikethroughDisabledFeeds">
         <property name="text">
          <string>Strikethrough disabled feed titles</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cbUpdateFeedListDuringFetching">
         <property name="text">
          <string>Update feed list during feed fetching</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabMessages">
      <attribute name="title">
       <string>Articles</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="m_checkRemoveReadMessagesOnExit">
         <property name="text">
          <string>Remove all read articles from all feeds on application exit</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cmbIgnoreContentsChanges">
         <property name="text">
          <string>Ignore changes in article body when new articles are being fetched</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cbFixupArticleDatetime">
         <property name="text">
          <string>Fixup date/time of articles which are in the future</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="m_checkBringToForegroundAfterMsgOpened">
         <property name="text">
          <string>Bring application window to front once article is opened in external web browser</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Internal article viewer</string>
         </property>
         <layout class="QFormLayout" name="formLayout_6">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="m_cbShowEnclosuresDirectly">
            <property name="text">
             <string>Display attachments directly in article</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="m_lblHeightImageAttachments">
              <property name="text">
               <string>Limit height of all pictures</string>
              </property>
              <property name="buddy">
               <cstring>m_spinHeightImageAttachments</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_spinHeightImageAttachments">
              <property name="minimum">
               <number>-1</number>
              </property>
              <property name="maximum">
               <number>4000</number>
              </property>
              <property name="value">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Article browser font</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="m_lblMessagesFont">
                 <property name="text">
                  <string>Font preview</string>
                 </property>
                 <property name="buddy">
                  <cstring>m_btnChangeMessagesFont</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="m_btnChangeMessagesFont">
                 <property name="text">
                  <string>&amp;Change font</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cbArticleViewerAlwaysVisible">
         <property name="text">
          <string>Keep article viewer always visible</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cbLegacyArticleFormatting">
         <property name="text">
          <string>Use legacy article formatting</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="m_cbUnreadWhenUpdated">
         <property name="text">
          <string>Mark existing article unread when its updated version is fetched from the feed</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabArticlesList">
      <attribute name="title">
       <string>Articles list</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Unread article icon type</string>
         </property>
         <property name="buddy">
          <cstring>m_cmbUnreadIconType</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="m_cmbUnreadIconType"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="m_checkKeppMessagesInTheMiddle">
         <property name="text">
          <string>Keep article selection in the middle of the article list viewport</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="m_checkMultilineArticleList">
         <property name="text">
          <string>Enable multiline items</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="HelpSpoiler" name="m_helpMultilineArticleList" native="true">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Row height</string>
         </property>
         <property name="buddy">
          <cstring>m_spinHeightRowsMessages</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="m_spinHeightRowsMessages">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="suffix">
          <string notr="true"> px</string>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Top/bottom row padding</string>
         </property>
         <property name="buddy">
          <cstring>m_spinPaddingRowsMessages</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="m_spinPaddingRowsMessages">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="suffix">
          <string notr="true"> px</string>
         </property>
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="m_checkMessagesDateTimeFormat">
         <property name="text">
          <string>Use custom date/time format</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="m_cmbMessagesDateTimeFormat">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="m_checkMessagesDateTimeFormatForDatesOnly">
         <property name="text">
          <string>Use custom date/time format for dates-only</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="m_cmbMessagesDateTimeFormatForDatesOnly">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="m_checkMessagesTimeFormat">
         <property name="text">
          <string>Custom date/time format for today's articles</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="m_cmbMessagesTimeFormat">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Show relative time for articles not older than</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSpinBox" name="m_spinRelativeArticleTime">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QGroupBox" name="m_gbArticleListFont">
         <property name="title">
          <string>Article list font</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="m_lblMessageListFont">
            <property name="text">
             <string>Font preview</string>
            </property>
            <property name="buddy">
             <cstring>m_btnChangeMessageListFont</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_btnChangeMessageListFont">
            <property name="text">
             <string>&amp;Change font</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QComboBox" name="m_cmbArticleMarkingPolicy"/>
         </item>
         <item>
          <widget class="QSpinBox" name="m_spinArticleMarkingPolicy">
           <property name="suffix">
            <string notr="true"> ms</string>
           </property>
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>15000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Upon article selection, mark as read</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="m_checkSwitchArticleListRtl">
         <property name="text">
          <string>Switch article list to RTL layout if all selected feeds are RTL</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>timespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>HelpSpoiler</class>
   <extends>QWidget</extends>
   <header>helpspoiler.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArticleAmountControl</class>
   <extends>QWidget</extends>
   <header>articleamountcontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_checkUpdateAllFeedsOnStartup</tabstop>
  <tabstop>m_spinStartupUpdateDelay</tabstop>
  <tabstop>m_checkAutoUpdate</tabstop>
  <tabstop>m_spinAutoUpdateInterval</tabstop>
  <tabstop>m_checkAutoUpdateOnlyUnfocused</tabstop>
  <tabstop>m_cmbFastAutoUpdate</tabstop>
  <tabstop>m_spinFeedUpdateTimeout</tabstop>
  <tabstop>m_spinHeightRowsFeeds</tabstop>
  <tabstop>m_btnChangeFeedListFont</tabstop>
  <tabstop>m_cmbCountsFeedList</tabstop>
  <tabstop>m_cbHideCountsIfNoUnread</tabstop>
  <tabstop>m_cbUpdateFeedListDuringFetching</tabstop>
  <tabstop>m_cbListsRestrictedShortcuts</tabstop>
  <tabstop>m_checkShowTooltips</tabstop>
  <tabstop>m_cbStrikethroughDisabledFeeds</tabstop>
  <tabstop>m_checkRemoveReadMessagesOnExit</tabstop>
  <tabstop>m_cbArticleViewerAlwaysVisible</tabstop>
  <tabstop>m_cmbIgnoreContentsChanges</tabstop>
  <tabstop>m_cbUnreadWhenUpdated</tabstop>
  <tabstop>m_cbFixupArticleDatetime</tabstop>
  <tabstop>m_checkBringToForegroundAfterMsgOpened</tabstop>
  <tabstop>m_cbShowEnclosuresDirectly</tabstop>
  <tabstop>m_spinHeightImageAttachments</tabstop>
  <tabstop>m_btnChangeMessagesFont</tabstop>
  <tabstop>m_cmbUnreadIconType</tabstop>
  <tabstop>m_checkKeppMessagesInTheMiddle</tabstop>
  <tabstop>m_checkSwitchArticleListRtl</tabstop>
  <tabstop>m_checkMultilineArticleList</tabstop>
  <tabstop>m_spinHeightRowsMessages</tabstop>
  <tabstop>m_spinPaddingRowsMessages</tabstop>
  <tabstop>m_checkMessagesDateTimeFormat</tabstop>
  <tabstop>m_cmbMessagesDateTimeFormat</tabstop>
  <tabstop>m_checkMessagesTimeFormat</tabstop>
  <tabstop>m_cmbMessagesTimeFormat</tabstop>
  <tabstop>m_spinRelativeArticleTime</tabstop>
  <tabstop>m_btnChangeMessageListFont</tabstop>
  <tabstop>m_gbFeedListFont</tabstop>
  <tabstop>m_gbArticleListFont</tabstop>
  <tabstop>m_tabFeedsMessages</tabstop>
  <tabstop>m_cbLegacyArticleFormatting</tabstop>
  <tabstop>m_checkMessagesDateTimeFormatForDatesOnly</tabstop>
  <tabstop>m_cmbMessagesDateTimeFormatForDatesOnly</tabstop>
  <tabstop>m_cmbArticleMarkingPolicy</tabstop>
  <tabstop>m_spinArticleMarkingPolicy</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
